### ISSUE TITLE
--shell causes "Unable to canonicalize the source directory. Reason: No such file or directory (os error 2)"

### ISSUE THREAD

**Issue by deadda7a** [stepchowfun/toast#435]
**Description**
When using the --shell or -s parameter to drop into a shell in the container after all the tasks have run, the "Unable to canonicalize the source directory. Reason: No such file or directory (os error 2)" error rises and execution stops.

**Instructions to reproduce the bug**
run toast with the -s parameter, it happens with every Toastfile, for example
```yaml
image: debian:stable-slim

tasks:
  test:
    command: |
      echo Hello
```

This is what happens:
```
deadda7a@rainbow ~ % toast -s -f test.toast
[INFO] Ready to run 1 task: test.
[INFO] Running task test…
Hello
[INFO] Preparing a shell…
[ERROR] Unable to canonicalize the source directory. Reason: No such file or directory (os error 2)
```

**Environment information:**
 - Toast version: 0.45.4
 - OS: Debian Sid

**Additional context**
I put Debian Sid in the OS info because I use it locally on my dev machine, but the same issue also exists on Debian Stable (11 at the time being)

Maybe this is related to issue #245?

Also thank you for this awesome tool :)

**Comment by stepchowfun** [comment]
Hi @deadda7a, thanks for filing this bug! I'm able to reproduce the bug, and I'm surprised it hasn't been discovered until now!

I found out the root cause and will have it fixed shortly. In the meantime, you should be able to work around it by running `toast -s -f ./test.toast` instead of `toast -s -f test.toast`.

**Comment by stepchowfun** [comment]
The fix has landed in v0.45.5. You can install it now using the installer script or from source. It may take some time to land in the various package managers, such as Homebrew.
